File: sqlUpdate.py
import MySQLdb
#Connecting to database with mysqldb
db = MySQLdb.connect(host="localhost", user="user", passwd="password", db="school")
#Creating cursor
cur = db.cursor(MySQLdb.cursors.DictCursor)
#Cursor executing sql code and commiting
sql = "UPDATE students SET age='17' WHERE name='Eric' AND age=16"
cur.execute(sql)
db.commit()
#Disconnecting cursor and database
cur.close()
db.close()

File: sqlUpdate.php
<?php
$servername = "localhost";
$username = "user";
$password = "password";
$dbname = "school";

//Create connection
$conn = new mysqli($servername, $username, $password, $dbname);
//Check connection
if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);
}
//Update Data
$sql = "UPDATE students SET gradeLevel='12' WHERE name='Eric' AND gradeLevel=11";

//Echos success or an error
if ($conn->query($sql) === TRUE) {
    echo "Record updated successfully";
} else {
    echo "Error updating record: " . $conn->error;
}

$conn->close();
?>
